scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -124,6 +128,35 @@
                       'Optional (defaults to True). If true, metrics will be collected from the container and reported  '
                       'to Scalyr.', convert_to=bool, default=True)
 
+define_config_option( __monitor__, 'label_include_globs',
+                     'Optional (defaults to [\'*\']). If `labels_as_attributes` is True then this option is a list of glob strings used to '
+                     'include labels that should be uploadeded as log attributes.  The docker monitor first gets all container labels that '
+                     'match a glob in this list and then filters out any labels that match a glob in `label_exclude_globs`, and the final list is then '
```

Comment:
I would replace with `match any glob in this list`

Reply:
Done.